src/ray/rpc/grpc_server.cc

```diff
@@ -50,6 +50,10 @@ void GrpcServer::Run() {
                              RayConfig::instance().max_grpc_message_size());
   builder.AddChannelArgument(GRPC_ARG_MAX_RECEIVE_MESSAGE_LENGTH,
                              RayConfig::instance().max_grpc_message_size());
+  builder.AddChannelArgument(GRPC_ARG_KEEPALIVE_TIME_MS, 30000);
+  builder.AddChannelArgument(GRPC_ARG_KEEPALIVE_TIMEOUT_MS, 30000);
```

Comment:
What are defaults? Also, will some of our long polling protocol work with this?

Reply:
Btw, setting these values the same sounds a bit dangerous? This means we will only have one ping for keepalive. check and then it will crash right?